apps/docs/app/globals.css

```
@@ -1,26 +1,259 @@
+/* ── CORNER DOTS ── */
+.corner-dots {
+  position: relative;
+}
+.corner-dots::before {
+  content: '';
+  position: absolute;
+  top: -8px;
+  left: -8px;
+  right: -8px;
+  bottom: -8px;
+  background: rgba(0,0,0,0.02);
```

Stylelint (17.10.0)

[error] Duplicate property "background" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)